.github/workflows/lint-test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           fetch-depth: 0
 
       - name: Set up Helm
-        uses: azure/setup-helm@v3.3.0
+        uses: azure/setup-helm@v3
```

Comment:
Probably rebase this branch on top of main. This change should not be included.

Reply:
We should lock on the patch release.

Reply:
Changed to `v3.3`